cypress/integration/page.spec.ts

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+/* eslint-disable max-len */
+import { faker } from '@faker-js/faker';
+import { peopleFromServer } from '../../src/data/people';
+import { Person } from '../../src/types/Person';
+
+enum DataQa {
+  Title = 'title',
+  SearchInput = 'search-input',
+  SuggestionsList = 'suggestions-list',
+  SuggestionItem = 'suggestion-item',
+  NoSuggestionsMessage = 'no-suggestions-message',
+}
+
+const getByDataQa = (id: DataQa) => cy.get(`[data-qa="${id}"]`);
```

Comment:
typically we use `data-cy` and already have a helper for it
https://github.com/mate-academy/fed/blob/master/cypress-tools/support/commands.js